Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ComponentEditor.hxx

```diff
@@ -101,7 +101,16 @@ namespace AzToolsFramework
 
         void SetComponentOverridden(const bool overridden);
 
-        void EnteredComponentMode(const AZStd::vector<AZ::Uuid>& componentModeTypes);
+        enum ComponentEditorState
+        {
+            DidEnterComponentMode,
+            DidNotEnterComponentMode
+        };
+
+        /// Disables the ComponentEditor if the componentModeType has not entered component mode.
+        /// Returns true and highlights the ComponentEditor widget if the componentModeType has enterered component mode.
+        ComponentEditorState EnteredComponentMode(const AZStd::vector<AZ::Uuid>& componentModeTypes);
```

Comment:
update: This comment isn't accurate I'm afraid - if we update the names we can say 'Returns the state of this specific component card when Component Mode is entered (inactive - that component isn't in component mode, active - that component is in component mode, selected - that component is in component mode and is highlighted/selected (orange outline).